Services/Concurrency/RatedCounter.cs

```diff
@@ -107,7 +110,7 @@ public long GetPause()
                 this.CleanQueue(now);
 
                 // No pause if the limit hasn't been reached yet,
-                if (this.timestamps.Count < this.eventsPerTimeUnit)
+                if (this.timestamps.Count < this.EventsPerTimeUnit)
```

Comment:
Nit: I think that our code style uses lower-case for the first character of variable names. I use ReSharper and it tells me that I'm out of compliance when I have variable names start with uppercase characters.